src/plot_api/plot_api.js

```diff
@@ -383,6 +383,7 @@ Plotly.plot = function(gd, data, layout, config) {
         drawFramework,
         marginPushers,
         marginPushersAgain,
+        initInteractions,
```

Comment:
just want to call out a note I put in a [commit message](https://github.com/plotly/plotly.js/pull/1767/commits/1570274c426dfda4cf3e57c312d7fa14b531a696):
```
I don't understand why we need to initInteractions twice,
but it seems necessary to do both before and after finalDraw
```

I'll have a perf PR coming after this merges and will look into that in more detail.